### PR TITLE
Updated angular app portal assets

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
In the pr, the images were replaced by svg images existing within the library

### CODE REVIEW

**Comment by horlah** [comment]
@Oluwadaminiola Please see if you can put these SVG files in a separate file and link it to the HTML document.

Please don't forget to delete any image file or folder that might exist in the dashboard-component project.

**Comment by Oluwadaminiola** [comment]
> @Oluwadaminiola Please see if you can put these SVG files in a separate file and link it to the HTML document.
> 
> Please don't forget to delete any image file or folder that might exist in the dashboard-component project.

okay

**Comment by horlah** [comment]
@Oluwadaminiola What's the update on this?

**Review by horlah — APPROVED** [review]

File: web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts [hunk 1]
```diff
@@ -265,6 +264,8 @@ export class ConvoyAppComponent implements OnInit {
 			});
 
 			this.appDetails = appDetailsResponse.data;
+			this.getEvents() 
+			this.getEventDeliveries()
```
Comment (horlah) [inline web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts:268]: This two shouldn't be called here

**Review by horlah — CHANGES_REQUESTED** [review]

File: web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts [hunk 2]
```diff
@@ -96,7 +95,7 @@ export class ConvoyAppComponent implements OnInit {
 	}
 
 	async initDashboard() {
-		await Promise.all([this.getEvents(), this.getEventDeliveries(), this.getAppDetails()]);
+		this.getAppDetails()
```
[the remark this answers is not in the document]
Comment (Oluwadaminiola) [reply web/components/app-portal-angular/projects/convoy-app/src/lib/convoy-app.component.ts:98]: okay

**Review by horlah — APPROVED** [review]
💪🏾

**Pull request merged**